front/app/components/admin/PostManager/components/FilterSidebar/FilterSidebarProjects.tsx

```diff
@@ -39,6 +41,20 @@ const FilterSidebarProjects = ({
           <Divider />
         </>
       )}
+      <Box display="inline-flex">
```

Comment:
Maybe I would create a shared component for these 3 buttons, but up to you of course.